Std/Data/Option/Basic.lean

```diff
@@ -84,6 +84,12 @@ inductive Rel (r : α → β → Prop) : Option α → Option β → Prop
   /-- `none ~ none` -/
   | none : Rel r none none
 
+/-- Lifts a relation `α → β → Prop` to a relation `Option α → Option β → Prop` by making
+`none` incomparable. -/
```

Comment:
```suggestion
`none` unrelated to anything. -/
```

Reply:
I'm not minded to resist this suggestion but me myself I don't really think it matters too much.

Reply:
Reviews aren't only about things that matter. Not changing the status to `awaiting-author` is what signaled that this is not important enough to block merging.

Reply:
For sure! I just wanted to say I was accepting the change because I do quite like it, but I'm probably not minded to pull my hair out trying to find the perfect wording.